docs-ref-conceptual/use-azure-cli-successfully-troubleshooting.md

```diff
@@ -183,7 +183,15 @@ or different zone. See https://aka.ms/azureskunotavailable for details."}]}}
 
 The solution is to change a property of your requested Azure resource, or try a different location.
 
-## Error: SSLError "bad handshake...certificate verify failed" (Proxy blocks connection)
+## Error: Subscription not found
+
+Assuming that you have not incorrectly typed a subscription name or ID, this error occurs when a resource provider is not registered in the active subscription. For example, if you want to execute `az storage account create`, the `Microsoft.Storage` provider must be registered. To register a resource provider, see [Azure resource providers and types](/azure/azure-resource-manager/management/resource-providers-and-types).
+
+## Error: Bad handshake...certificate verify failed
+
+See [Work behind a proxy](#work-behind-a-proxy) for information on how to resolve this error.
```

Comment:
Any source stating using H2 can give Search Engine Optimization (SEO)? Why doesn't putting the error in the paragraph have this advantage?

Reply:
@jiasli, paragraphs aren't indexed.  Try these two searches on our troubleshooting doc:

**H2:**
`azure cli common syntax errors`

**Paragraph:**
`azure cli errors that come from incorrect quoting`

**Bing copilot:** (returns in-paragraph text)
`azure cli errors that come from incorrect quoting`

Did I understand your question correctly?